content/en/security/detection_rules/_index.md

```diff
@@ -48,7 +48,7 @@ To [create custom rules](#create-detection-rules), you can clone the default rul
 
 ## Search and filter detection rules
 
-To view out-of-the-box and custom detection rules in Datadog, navigate to the [**Security** > **Configuration**][15] page. Rules are listed on separate pages for each product (Application Security, Cloud Security Management, and Cloud SIEM).
+To view out-of-the-box and custom detection rules in Datadog, navigate to the [**Security Settings**][15] page. Rules are listed on separate pages for each product (Application Security, Cloud Security Management, and Cloud SIEM).
```

Comment:
This link sends me to https://app.datadoghq.com/security/configuration/notification-rules
which doesn't match the Cloud SIEM image, should it link to https://app.datadoghq.com/security/configuration/siem/rules?query=type%3A%28log_detection%20OR%20signal_correlation%29&deprecated=hide&groupBy=tactic&sort=rule_name ?